### DISCUSSION TITLE
ImageError: "url" parameter is valid but upstream response is invalid

### DISCUSSION THREAD

**Discussion by nicitaacom** [vercel/next.js#64028] [Help] [answered]
### Link to the code that reproduces this issue

https://codesandbox.io/p/devbox/mvp-image-error-upstream-response-is-invalid-rfwdwv

### To Reproduce

Similar issue - https://github.com/vercel/next.js/issues/53715
Still not resolved - you can make sure in that by scrolling down and find comment Nov 9 that this issue still persist in Next 14

Steps to reproduce:
1. Open [this](https://codesandbox.io/p/devbox/mvp-image-error-upstream-response-is-invalid-rfwdwv) codesandbox
2. See error in console

![image](https://github.com/vercel/next.js/assets/39565703/69467141-cff6-4975-8ab8-d8de30b9ef7c)

### Current vs. Expected behavior

I'm expecing no error
I see error

### Provide environment information

```bash
Operating System:
  Platform: linux
  Arch: x64
  Version: #1 SMP PREEMPT_DYNAMIC Sun Aug  6 20:05:33 UTC 2023
Binaries:
  Node: 20.11.1
  npm: 10.2.4
  Yarn: 1.22.19
  pnpm: 8.15.4
Relevant Packages:
  next: 14.1.4
  eslint-config-next: 14.1.4
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.4.3
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

Image optimization (next/image, next/legacy/image)

### Which stage(s) are affected? (Select all that apply)

next dev (local)

### Additional context

If you run into similar issue - please vote up

**Comment by balazsorban44** [comment]
It looks like the upstream image is unavailable (gives 404). https://codesandbox.io/p/devbox/mvp-image-error-upstream-response-is-invalid-rfwdwv?file=%2Fapp%2Fpage.tsx%3A8%2C26

It returns this response 
```json
{
  "statusCode": "404",
  "error": "not_found",
  "message": "Object not found"
}
```

hence it cannot be rendered. make sure the upstream URL is pointing to an existing image

**Comment by nicitaacom** [comment, marked as answer]
> It looks like the upstream image is unavailable (gives 404). 

Agreed

![image](https://github.com/vercel/next.js/assets/39565703/f86b4458-60b5-41d1-9f33-3e87f822fdc9)
